manifests/07-operator.yaml

```diff
@@ -31,7 +31,7 @@ spec:
         - console
         - operator
         args:
-        - "-v=4"
+        - "-v=100"
```

Comment:
don't do this.  it just made your log unreadable and doesn't aid debugging.

Reply:
agreed!  never meant to be long term, but thanks!